samples/MyLibrary/Class1.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Hosting.Internal;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Localization;
+
+namespace MyLibrary
+{
+    public class Class1
+    {
+        private IStringLocalizer<Class1> _localizer;
+
+        public Class1()
+        {
+            var services = new ServiceCollection();
```

Comment:
You shouldn't do something artificial like this, specially if it's on a sample. Follow the patterns that we would recommend a customer to follow.